public/scripts/getissdata.js

```diff
@@ -0,0 +1,10 @@
+//Get ISS data from Open Notify and pass it to the url.
+const button = document.getElementsByClassName('showbtn')[0];
+button.addEventListener('click', () => {
+  fetch(`http://api.open-notify.org/iss-now.json`).then(response => {
```

Comment:
The indentation here seems different from the rest of the app

Reply:
Sure. I've just installed it and will be making changes asap.